src/server/routes/merge.ts

```diff
@@ -164,19 +183,30 @@ function loadEntityRelationships(entity, orm, transacting): Promise<any> {
 			withRelated: [
 				'relationships.source',
 				'relationships.target',
-				'relationships.type'
+				'relationships.type.attributeTypes',
+				'relationships.attributeSet.relationshipAttributes.value',
+				'relationships.attributeSet.relationshipAttributes.type'
 			]
 		})
 		.then((relationshipSet) => {
 			entity.relationships = relationshipSet ?
 				relationshipSet.related('relationships').toJSON() : [];
 
+			// Attach attributes to relationship object
+			entity.relationships.forEach((relationship) => {
+				if (relationship.attributeSet?.relationshipAttributes) {
+					relationship.attributeSet.relationshipAttributes.forEach(attribute => {
+						relationship[`${attribute.type.name}`] = attribute.value.textValue;
+					});
+				}
+			});
+
```

Comment:
This is also duplicated code from /src/server/helpers/middleware.ts , it would be better to make a shared utility function